sql_generators/active_users/templates/desktop_checks.sql

```diff
@@ -94,8 +94,17 @@ live_table_qdau_count AS (
 )
 SELECT
   IF(
-    (SELECT * FROM live_table_qdau_count) <> (SELECT * FROM qdau_sum),
-    ERROR("QDAU mismatch between aggregates table and live table"),
+    ABS((SELECT * FROM qdau_sum) - (SELECT * FROM live_table_qdau_count)) > 10,
+    ERROR(
+      CONCAT(
+        "QDAU mismatch between aggregates table and live table is greated than 10.",
+        " (live: ",
```

Comment:
total here instead of live might be confusing, total of what? I did follow one of your previous suggestions and and try to indicate the source tables. Unfortunately, since multiple tables are used the best I can do is to try and provide the app_name and the table name. Now the message looks like this:
```"DAU mismatch between the firefox_ios `baseline_v1` live (all channels) and active_users_aggregates (`firefox_ios_derived.active_users_aggregates_v2`) tables is greated than 10."```